app/src/main/java/org/schabi/newpipe/player/BackgroundPlayer.java

```diff
@@ -193,18 +201,51 @@ private void resetNotification() {
         setupNotification(notRemoteView);
         setupNotification(bigNotRemoteView);
 
-        NotificationCompat.Builder builder = new NotificationCompat.Builder(this, getString(R.string.notification_channel_id))
-                .setOngoing(true)
-                .setSmallIcon(R.drawable.ic_newpipe_triangle_white)
-                .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
-                .setCustomContentView(notRemoteView)
-                .setCustomBigContentView(bigNotRemoteView);
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(this, getString(R.string.notification_channel_id));
+        builder.setOngoing(true);
+        builder.setSmallIcon(R.drawable.ic_newpipe_triangle_white);
+        builder.setVisibility(NotificationCompat.VISIBILITY_PUBLIC);
+        builder.setCustomContentView(notRemoteView);
+        builder.setCustomBigContentView(bigNotRemoteView);
```

Comment:
Why did you convert the method call chain into single calls? Afaik it is good practice to arrange methods in a chain when using Builders/Factories.

Reply:
good question. I think we should undo this, too

Reply:
Yeah, there was no reason for me to do it like this. Reverted it to how it was initially.